prereise/gather/demanddata/transportation_electrification/const.py

```diff
@@ -36,4 +36,5 @@
 
 safety_coefficient = 1
 
+# ???
```

Comment:
What is that?

Reply:
Just a note to myself that we need to ask what the `ER` variable is?